docs/source/example.rst

```diff
@@ -0,0 +1,119 @@
+.. _example:
+
+Examples
+========
+
+First of all, you can install mirdata using `pip`.
+
+.. code-block:: bash
+    :linenos:
+
+    $ pip install mirdata
+
+With `mirdata`, the first thing you would do is to download the dataset.
+Depending on the availability of the audio files, `mirdata` may only show you
+an instruction about how to download the dataset.
+Fortunately, we can download Orchset dataset directly.
+
+
+.. code-block:: python
+    :linenos:
+
+    import mirdata
+    # Download the Orchset Dataset
+    mirdata.orchset.download()
+    # Orchset_dataset_0.zip?download=1: 1.00B [03:05, 185s/B]
+
+
+Once downloading is done, you can find the the dataset folder.
+
+.. code-block:: bash
+    :linenos:
+
+    $ ls ~/mir_datasets/Orchset/
+    GT
+    Orchset - Predominant Melodic Instruments.csv
+    README.txt
+    audio
+    midi
+
+The ID's and annotation data can be loaded as below.
+
+.. code-block:: python
+    :linenos:
+
+    # Load the dataset
+    orchset_data = mirdata.orchset.load()
+
+    orchset_ids = mirdata.orchset.track_ids()
+    orchset_data = mirdata.orchset.load()
+
+    # todo: add __str__() method and print(orchset_data)
+
+
+A more end-to-end example would be like this.
```

Comment:
"If we wanted to use Orchset to evaluate the performance of a melody extraction algorithm (in our case, `very_bad_melody_extractor`), and then split the scores based on the metadata, we could do the following:"